src/Sarif.Viewer.VisualStudio/ErrorListService.cs

```diff
@@ -163,9 +163,11 @@ private void WriteRunToErrorList(Run runLog)
                         HelpLink = rule?.HelpUri?.ToString()                        
                     };
 
-                    CaptureAnnotatedCodeLocationCollections(result.Stacks, AnnotatedCodeLocationKind.Stack, sarifError);
+                    IEnumerable<IEnumerable<AnnotatedCodeLocation>> stackLocations = CreateAnnotationsFromStacks(result.Stacks);
+
```

Comment:
Below you use a method named `CaptureAnnotatedCodeLocationCollections`. I think this one should be named `CreateAnnotatedCodeLocationCollections`. (and I don't think you need "FromStacks" since the argument tells you that).